app/views/katello/api/v2/content_view_filter_rules/show.json.rabl

```diff
@@ -12,5 +12,5 @@ attributes :errata_id, :if => lambda { |rule| rule.respond_to?(:errata_id) && !r
 attributes :start_date, :if => lambda { |rule| rule.respond_to?(:start_date) && !rule.start_date.blank? }
 attributes :end_date, :if => lambda { |rule| rule.respond_to?(:end_date) && !rule.end_date.blank? }
 attributes :types, :if => lambda { |rule| rule.respond_to?(:types) && !rule.types.blank? }
-
+attributes :date_type, :if => lambda { |rule| rule.respond_to?(:date_type) }
```

Comment:
Nitpick - could you move this to line 15 and keep the blank line separating the attributes from the extends?